mvc/views/tag-helpers/authoring.rst

```diff
@@ -1,8 +1,394 @@
+ 
+When multiple attributes are added to the same statement, the runtime treats them as a logical-AND. For example, in the code below, an HTML element must be named "bold" with an attribute named "bold" ( <bold bold /> ) to match.
+
+.. code-block:: c#
+
+        [TargetElement("bold", Attributes = "bold")]
+
+For a good example of a bootstrap progress bar that targets a tag and an attribute, see `Creating custom MVC 6 Tag Helpers <http://www.davepaquette.com/archive/2015/06/22/creating-custom-mvc-6-tag-helpers.aspx>`__.
+
+You can also use the ``[TargetElement]`` to change the name of the targeted element. For example if you wanted the ``BoldTagHelper`` to target ``<MyBold>`` tags, you would use the following attribute:
+
+.. code-block:: c#
+
+	    [TargetElement("MyBold")]
+
+Web site information Tag Helper
+------------------------------------
+
+#. Add a *Models* folder.
+#. Add the following ``WebsiteContext`` class to the *Models* folder:
+
+.. literalinclude:: authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Models/WebsiteContext.cs
+    :language: c#
+
+3. Add the following ``WebsiteInformationTagHelper`` class to the *TagHelpers* folder.
+
+.. literalinclude:: authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/TagHelpers/WebsiteInformationTagHelper.cs
+    :language: c#
+
+**Notes:** 
+
+- As mentioned previously, the ASP.NET 5 runtime translates Pascal-cased C# class names and properties for tag helpers into `lower kebab case <http://c2.com/cgi/wiki?KebabCase>`__. Therefore, to use the ``WebsiteInformationTagHelper`` in Razor, you'll write ``<website-information />``.
+- We are not explicitly identifying the target element with the ``[TargetElement]`` attribute, so the default of ``website-information`` will be targeted. If you applied the following attribute (note it's not kebab case but matches the class name):
+
+.. code-block:: c#
+
+	    [TargetElement("WebsiteInformation")]
+
+The lower kebab case tag ``<website-information />`` would not match. If you want use the ``[TargetElement]`` attribute, you would use kebab case as shown below:
+
+.. code-block:: c#
+
+	    [TargetElement("Website-Information")]
+
+- Elements that are self-closing have no content. For this example, the Razor markup will use a self-closing tag, but the tag helper will be creating a `section <http://www.w3.org/TR/html5/sections.html#the-section-element>`__ element (which is not self-closing and we are writing content inside the ``section`` element). Therefore, we need to set ``TagMode`` to ``StartTagAndEndTag`` to write output. Alternatively, you can comment out the line setting ``TagMode`` and write markup with a closing tag. (Example markup is provided later in this tutorial.)
+
+5. Add the following markup to the *About.cshtml* view. The highlighted markup displays the web site information.
+
+.. literalinclude::  authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/Home/About.cshtml
+   :language: aspx-cs
```

Comment:
AFAIK    :language: aspx-cs is currently ignored. I'm hoping it will be supported someday. @danroth27 can confirm.